wp-content/themes/hueman_child/style.css

```diff
@@ -38,4 +38,8 @@ License URI: http://www.gnu.org/licenses/gpl-3.0.html
 
 #nav-header.nav-container {
   background-color: #666;
+}
+
+.entry.excerpt { 
+  color: #666;

```

Comment:
Does this change effect footer issue?
In a PL, changes should be related to the same issue